mesh.yaml

```diff
@@ -143,7 +143,7 @@ processing:
         topic: outboxes/web-resources
     environment:
       # local test domain, see readme
-      STREAMX_BLUEPRINTS_SITEMAP-GENERATOR-PROCESSING-SERVICE_BASE-URL: "http://puresight.127.0.0.1.nip.io"
+      STREAMX_BLUEPRINTS_SITEMAP-GENERATOR-PROCESSING-SERVICE_BASE-URL: "http://puresight.${STREAMX_INGRESS_IP}.nip.io"
```

Comment:
Maybe we could configure whole domains, so that many people can deploy on the single cluster in multiple namespaces?